lidar_topic: /livox/lidar
imu_topic: /livox/imu

map_frame: lidar
body_frame: body

scan_resolution: 0.1
map_resolution: 0.5
max_point_thresh: 100
update_point_thresh: 20
estimate_ext: false
plane_thresh: 0.01

p_il: [ -0.011, -0.02329, 0.04412 ]